### DISCUSSION TITLE
请问哪个版本的支持导出的是xlsx而不是xls

### DISCUSSION THREAD

**Discussion by h742004725** [jxlsteam/jxls#319] [Q&A] [answered]
现在需要单个sheet导出超过65535条数据，目前使用的是2.10.0版本的，但是导出是用的xls，超过数量会报错，能否告知哪个版本的可以excel导出是用的xlsx

Now I need to export more than 65535 data items from a single sheet. I am currently using version 2.10.0, but the export is done using xls. If the number exceeds, an error will be reported. Can you tell me which version can use xlsx for excel export?

**Comment by SoltauFintel** [comment, marked as answer]
A XLSX file can contain 1 million rows.

Jxls 2.10.0 supports hssf, xssf and sxssf. Or in other words: .xls, .xlsx and .xlsx-with-streaming. Streaming is often used if you have many rows.

Create a XLSX template (from scratch?) using Microsoft Excel. You must use the xssf or sxssf API. I don't know how to do that off the top of my head.

My advice: Use the newest Jxls version 3.x and read the new docs. Streaming is explained there.
